src/base/promptElements.tsx

```diff
@@ -133,6 +133,11 @@ export interface TextChunkProps extends BasePromptElementProps {
 	breakOnWhitespace?: boolean;
 }
 
+export interface ImageProps extends BasePromptElementProps {
+	src: string;
```

Comment:
We should have a docblock explaining how to provide this (i.e. the base64 encoding)